Fix infra scripts (#373)
feltcoop/felt-server: src/infra/README.md

gro deploy

# TODO support custom deployment URLs

```

## manually upgrading cloud instances

If upgrading your underlying cloud OS, we recommend spinning up a new server instance and restoring a DB backup to it. This helps keeps cruft from forming on your servers, helps you make sure you running on the latest hardware from your cloud provider, and tests your DB restoration process all in one.

1. Spin up a new server with the new OS
1. Point your DNS record to the new IP
1. Run `infra/setup` on the new server
1. Set up your DB password

> 1. Take a dump of the DB from your old server instance

Reviewer: ```suggestion
1. Use [`pg_dump`](https://www.postgresql.org/docs/current/backup-dump.html) to copy the DB from your old server instance
```

phrasing :no_mouth:
